js/script.js

```diff
@@ -17,3 +16,152 @@ navMenu.forEach((links) => links.addEventListener('click', () => {
 }));
 
 window.onresize = () => { if (window.innerWidth > 760) navList.classList.remove('show'); };
+
+const projectsLists = [
+  {
+    name: 'Keeping track of hundreds  of components website',
+    image: 'assets/mobile/SnapshootPortfolio.svg',
+    imageDescription: 'Snapshoot portfolio',
+    description:
+    ["Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s."],
+    techList: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Multi-Post Stories',
+    image: 'assets/mobile/ImgPlaceholder-desk.svg',
```

Comment:
- [ ] This image is missing from the modal pop-up display for both mobile and desktop version. Kindly fix it. 👍